index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Personal Portfolio Website</title>
+</head>
+
+<body>
+    <header>
+        <nav class="navigation">
+            <div class="logo">
+                <h3>Dhuruv</h3>
+            </div>
+            <div class="menu">
+
+            </div>
+        </nav>
+    </header>
+    <main>
+        <div class="headline-section">
+            <div class="illustration-mobile">
+                <img src="./icons/header-illsutration-mobile@2x.png" alt="">
+            </div>
+            <div class="heading-text">
+                <h1>Hey there. I’m Dhuruv</h1>
+                <h1 class="h-green">I’m a software developer</h1>
```

Comment:
- [ ] Please there should be only one `h1` element in the document make sure to use other `hx` tags here instead this will make a role in SEO.

Reply:
I used span in the same one h1 instead of other hx